### DISCUSSION TITLE
Question about (custom) exports from a users own timesheet

### DISCUSSION THREAD

**Discussion by Jottger** [kimai/kimai#4941] [General]
Hey there,

i have created two custom pdf exports to be used with the dedicated export function via the menu on the left. They are working well. Now i decided to remove the default export templates to have a better overview in that menu (I used the _defaults: ~_ option in the local.yaml).
That worked, but now i can't use the default pdf-export via a users own timesheet in the top right. It just returns a critical error 500. It works when i enable the default templates again.

Is this intended? If so, is there a way to use custom exports via a users own timesheet? The documentation only covers the normal way of exporting data via the dedicated export menu.

**Comment by kevinpapst** [comment]
Use the default filenames that Kimai uses, then your file should be loaded.
